__tests__/ain.test.ts

```diff
@@ -102,6 +102,11 @@ describe('ain-js', function() {
       expect(afterLength).toBe(beforeLength);
     });
 
+    it('addAndSetDefaultAccount', function () {
+      ain.wallet.addAndSetDefaultAccount(test_sk);
+      expect(ain.wallet.defaultAccount).not.toBe(null);
```

Comment:
Should we check the default account did change to the account corresponding to test_sk?

Reply:
Looks good! Done